frontend/src/__tests__/cypress/cypress/pages/modelRegistry.ts

```diff
@@ -39,6 +73,42 @@ class ModelRegistry {
     appChrome.findNavItem('Model Registry').should('not.exist');
     return this;
   }
+
+  findTable() {
+    return cy.findByTestId('registered-model-table');
+  }
+
+  findTableRows() {
+    return this.findTable().find('tbody tr');
+  }
+
+  getRow(name: string) {
+    return new RegisteredModelTableRow(() =>
+      this.findTable().find(`[data-label="Model name"]`).contains(name).parents('tr'),
+    );
+  }
+
+  findRegisteredModelTableHeaderButton(name: string) {
+    return this.findTable().find('thead').findByRole('button', { name });
+  }
+
+  findModalSearchInput() {
+    return cy.findByTestId('label-modal-search');
+  }
+
+  shouldContainsModalLabels(labels: string[]) {
+    cy.get('[data-testid="modal-label-group"]').within(() =>
+      labels.map((label) => cy.contains(label)),
+    );
+  }
```

Comment:
`should` functions `return this;`

Use `cy.findByTestId` instead of attribute selector.